cli/build.rs

```diff
@@ -70,13 +70,13 @@ fn main() {
   let bundle_path = o.join("COMPILER_SNAPSHOT.js");
   let snapshot_path = o.join("COMPILER_SNAPSHOT.bin");
   let mut custom_libs: HashMap<String, PathBuf> = HashMap::new();
+  custom_libs
+    .insert("lib.window.d.ts".to_string(), c.join("js/lib.window.d.ts"));
+  custom_libs
+    .insert("lib.worker.d.ts".to_string(), c.join("js/lib.worker.d.ts"));
```

Comment:
If this isn't 100% the same as `web_worker` I think we should indicate that it is specifically Deno's version... `lib.deno_worker.d.ts`?

Reply:
The other convention that is used in TS libs, would be `js/lib.deno.worker.d.ts`, which is actually maybe better.

Reply:
Not sure about it, we still might want to provide `new Deno.Worker()` which would essentially be Web Worker but with all bells and whistles in `Deno` namespace

Reply:
Which would essentially be `lib.worker.d.ts` with addition to `Deno` namespace on `self`